metrology/instruments/timer.py

```diff
@@ -34,6 +36,7 @@ def update(self, duration):
         if duration >= 0:
             self.meter.mark()
             self.histogram.update(duration)
+            self._total_time += duration
```

Comment:
``histogram`` keep this internal as ``self.sum``, we might just need to expose it properly.